content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+   :guilabel:`Model` name in the backend. Search (using front-end name) for this model in the
+   :guilabel:`Applies to` drop-down menu in the WhatsApp template.
+
+Often when changing the model or :guilabel:`Applies to` field the :guilabel:`Phone Field` made
+produce an error. The :guilabel:`Phone field `should always result in `Phone` or `Mobile`. To search
+the available fields, type in the front-end name (on user-interface) of the field in the search box.
+This will find a result from all of the available fields for the model (:guilabel:`Applies to`) that
+the template is created for. There may be multiple levels that need to be configured.
+
+Change the :guilabel:`Category` to fit either a :guilabel:`Marketing`, :guilabel:`Utility`, or
+:guilabel:`Authentication` category. Specifying an incorrect category can cause a flag from Meta In
+the approval process.
+
+Add and :guilabel:`User` that are allowed to use this template. On the right hand column of the top
+configurations a :guilabel:`Header type` can be configured along with a :guilabel:`Header message`.
+
+The available :guilabel:`Header types` are as follows:
+
+- Text
+- Image
+- Video
+- Document
+- Location
+
+Navigate to the :guilabel:`Body` tab to configure the main message of the template.
+
+When all the necessary changes are made to the template, then click on the :guilabel:`Submit for
+approval` button in the upper left hand corner. The status of the template will change to
+:guilabel:`Pending`. The status will remain in :guilabel:`Pending` until a decision has been made by
+Meta. Once the user receives an email confirming that the template has been approved the templates
+will need to be synced from the Odoo database.
+
+See this documentation: :ref:`whatsapp/sync`.
+
+.. tip::
+   Pre-configured demo-data templates can be submitted to Meta for approval. Navigate to
+   :menuselection:`WhatsApp app --> Templates` and select the pre-configured template. Click
+   :guilabel:`Submit for Approval` to start the approval process. An email will be sent to the
+   administrator of the Meta account when the template has been approved.
+
+Using placeholders and variables
+--------------------------------
+
+Dynamic variables reference certain fields within the Odoo database to produce unique data in the
+WhatsApp message when using a template. Dynamic variables are encoded to display fields from within
+the database referencing fields from within a model.
+
+.. example::
+   Many companies like to customize their WhatsApp messages with a personalized piece of customer
+   information to grab attention. This can be accomplished in Odoo by referencing a field within a
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` field on the :guilabel:`Sales Order` model.
+
+Dynamic variables can be added in to the :guilabel:`Body` by adding :guilabel:`placeholders` in the
```

Comment:
```suggestion
At the bottom of the template  form, there are three tabs, :guilabel:`Body`, :guilabel:`Buttons`, and :guilabel:`Variables`. These three tabs combined create the template. The text is entered into the :guilabel:`Body` tab, and dynamic content that is called out in the :guilabel:`Body` tab is specified in the :guilabel:`Variables` tab. Every piece of dynamic content (variables) in the message (body) is specifically called out and specified in the :guilabel:`Variables` tab.

For example, in the body tab, if the following is typed in the body tab: "Hello {{1}}," , then "{{1}}" must be described in the variables tab, and the information that needs to be pulled to populate the field labeled {{1}} must be configured in the variables tab. For this specific case, the message should greet the customer by name, so the {{1}} shoul dbe configured to populate the {{1}} filed with the :guilabel:`User Name`.

Dynamic variables can be added in to the :guilabel:`Body` by adding :guilabel:`placeholders` in the
```


I am not 100% sure this works with the following 2 sentences, but I think it is very important to make it clear that the text is calling out items form the other tab (it took e a minute to figure this out). Plus, it would benefit form an image with the tabs shown. Or maybe with both tabs side by side, and an arrow pointing out the configured info for one of the placeholders. It's not a very complicated concept but for someone first seeing it, like me, it could be very confusing at first. This might help.

Reply:
Resolved, added modification of this in.